index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="app-wrap">
+        <div class="header-tab">
```

Comment:
- [ ] The header should be wrapped inside a `<header>` tag, both for both for semantic and search engine optimizazion reasons.

Reply:
Is the header the toolbar only or the whole page I made?

Reply:
> Is the header the toolbar only or the whole page I made?

It is only the first section
![](https://i.imgur.com/jSGtcRA.png)

Reply:
Okay thank you.